### PR TITLE
Fixed bug from issue  #757 , added showdown library and converted markdown to html in workbench.js

### PR DESCRIPTION
**Pull request by hannyP**
### fixed the bug

 I am an outreachy 2024 applicant and found this issue while running the project on my local machine  

## Testing
it hasn't been tested as only UI elements were being tested in the repo.

## Questions
I used the showdown library to solve this issue , let me know if i shouldn't have used any library do tackle this issue

### CODE REVIEW

**Comment by hannyP** [comment]
now when you click on help in workbench tab it should display the user guide like this
Screenshot 2024-03-13 164941

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (birm) [inline .vscode/settings.json:1]: Please don't include setting/config files for editors in PRs.

**Review by birm — CHANGES_REQUESTED** [review]
Just one change and I think It'll look good!

File: .vscode/settings.json [hunk 2]
```diff
@@ -0,0 +1,3 @@
...
+    "liveServer.settings.port": 5501
+}
```
Comment (birm) [inline .vscode/settings.json:3]:
```suggestion
```

**Comment by birm** [comment]
turns out to be fixed with #683

**Pull request closed without merging**